### PR TITLE
chore(release): bump to 5.0.0-beta.6

### PR DESCRIPTION
**Pull request by shlokamin**
# Overview

Bump to 5.0.0-beta.6

# Changelog

- bumped versions to 5.0.0-beta.6

# Review requests

-  Changlelogs look reasonable?

# Risk assessment

low

### CODE REVIEW

**Review by b-cooper — APPROVED** [review]
🐟

File: api/release-notes.md [hunk 1]
```diff
@@ -32,7 +32,6 @@ This beta is a work in progress! We'll be addressing the following features and
 
 - Your last run protocol will not be saved through robot reboots
 - You may experience various performance issues and freezes
-- The OT-2 will not drop any tips if a protocol is canceled
```
Comment (sanni-t) [inline api/release-notes.md:35]: Do we want to update the issue tracker with a `fixed-in-v6` label?

File: app-shell/build/release-notes.md [hunk 2]
```diff
@@ -26,13 +26,6 @@ This is beta software! You may experience unexpected crashes or missing features
 
 Keep an eye on this space as we release new versions of the beta. See the [issue tracker][] for a full list of issues resolved by the v5 release.
```
Comment (sanni-t) [inline app-shell/build/release-notes.md]: Update for v6?

**Comment by shlokamin** [comment]
Due to a critical bug we found homing the robot on cancel, we will not be releasing a `beta.6` just yet

**Pull request closed without merging**